src/parsing.cpp

```diff
@@ -97,13 +97,19 @@ void parse_env(std::shared_ptr<t_config> &config) {
       config->digest = false;
     }
   }
+
+  // Pipeline
+  config->pipeline = DEFAULT_GST_PIPELINE;
+  if (const char *pipeline = std::getenv("GST_PIPELINE")) {
+    config->pipeline = pipeline;
+  }
 }
 
 // Overwrite default parameters via cmd line
 bool parse_args(std::shared_ptr<t_config> &config, int argc, char **argv) {
   int c;
   opterr = 0;
-  while ((c = getopt(argc, argv, "r:u:l:p:b:f:s:i:ht")) != -1) {
+  while ((c = getopt(argc, argv, "r:u:l:p:b:f:s:i:htg:")) != -1) {
```

Comment:
please move the `g:` before `ht` (it's a tiny detail but it makes it easier to read when arguments are separated by whether or not they need a parameter

Reply:
make sense!